scaling/plugins/slurm.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+import shlex
+import logging
+import json
+
+log = logging.getLogger(__name__)
+
+# get nodes requested from pending jobs
+def pending(instance_type):
+    command = "squeue -r -h -o '%t %D'"
+    return run_command(command, 'PD')
+
+# get nodes requested from pending jobs
+def running(instance_type):
+    command = "squeue -r -h -o '%t %D'"
+    return run_command(command, 'R')
+
+def run_command(command, status):
+    args = shlex.split(command)
+    pending = 0
+    try:
+       _output = subprocess.check_output(args)
+       output = _output.split("\n")
+       for line in output:
+          line_arr = line.split()
+          if len(line_arr) == 2 and line_arr[0] == status:
+             pending += int(line_arr[1])
```

Comment:
Please rename it, see comment for 'torque'.

Reply:
changed.